public/global.css

```diff
@@ -7,6 +7,7 @@
     --lightbrand-color: #FF9191;
     --accent-color: #8DCAEF;
     --darkgrey: #B2B2B2; /*outline*/
+    --outline: #EEEEEE;
```

Comment:
This is the border for white cells on subtle background just to give a little pop. Through the shared `cell` class this affects search results list items, and quiz cards.